folly/experimental/logging/example/main.cpp

```diff
@@ -16,7 +16,7 @@
 #include <folly/experimental/logging/Init.h>
 #include <folly/experimental/logging/xlog.h>
 #include <folly/init/Init.h>
-
+#include <folly/portability/GFlags.h>
```

Comment:
Style nit. Please leave two groups of includes as before, separated by a blank line. This added include should be in the first group.